onnxruntime/test/providers/cpu/tensor/upsample_op_test.cc

```diff
@@ -173,7 +174,7 @@ TEST(UpsampleOpTest, UpsampleOpNearest222XTest) {
   };
 
   test.AddOutput<float>("Y", {N*2, C, (int64_t)(H * scales[2]), (int64_t)(W * scales[3])}, Y);
-  test.Run(OpTester::ExpectResult::kExpectSuccess, "", {kTensorrtExecutionProvider});//TensorRT parser: Assertion failed: scales[0] == 1 && scales[1] == 1
+  test.Run();//TensorRT parser: Assertion failed: scales[0] == 1 && scales[1] == 1
```

Comment:
should these assertion failure comments be removed if we are re-enabling the TensorrtExecutionProvider for these tests?

Reply:
same comment for other tests.

Reply:
removed the comments